### PR TITLE
Issue #KAIROS-381

### PR DESCRIPTION
**Pull request by kezhu**
pagination CLI

### CODE REVIEW

File: src/Pagination/PaginationCommand.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            $client = $this->getClient();
+            $page_size = $input->getOption('size');
+            $page_num_max = $input->getOption('pages');
+
+            if ($page_size > 25) {
+                $output->writeln("$page_size exceeds page size limit 25, setting it to 25 ...");
+                $page_size = 25;
+            }
+
+            $url = self::ENDPOINT_LIST."?limit=$page_size";
```
Comment (dstevenson) [inline src/Pagination/PaginationCommand.php]: We should really avoid building queries by hand. The client handles resolving the Uri components for us, we should be passing the options through to the guzzle method correctly

`->get($endpoint, $options)`

where `$options` is `['query' => ['limit' => $page_size]]`

Comment (treyhyde) [inline src/Pagination/PaginationCommand.php]: yes, as Doug said

File: bin/cd.php [hunk 2]
```diff
@@ -9,13 +9,14 @@
...
 use CentralDesktop\API\Auth\AuthCommand;
+use CentralDesktop\API\Pagination\PaginationCommand;
 
 $application = new Application();
 $application->add(new AuthCommand());
 $application->add(new WorkspacesCommand());
 $application->add(new UsersCommand());
 $application->add(new UploadFileCommand());
-
+$application->add(new PaginationCommand());
```
Comment (treyhyde) [inline bin/cd.php]: Is this really a top level function or should it be a set of args added to all commands?

Comment (dstevenson) [inline bin/cd.php]: agreed

File: bin/cd.php [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
-use CentralDesktop\API\Pagination\PaginationCommand;
+use CentralDesktop\API\Times\TimesCommand;
 
 $application = new Application();
 $application->add(new AuthCommand());
 $application->add(new WorkspacesCommand());
 $application->add(new UsersCommand());
 $application->add(new UploadFileCommand());
-$application->add(new PaginationCommand());
+$application->add(new TimesCommand());
```
Comment (kezhu) [inline bin/cd.php:19]: /v1/workspaces is currently broken on prod and /v1/users doesn't have the latest pagination mechanism so I added a /v1/times which works as expected.

Comment (osbornk) [inline bin/cd.php:19]: /v1/workspaces is a bad endpoint. Hopefully nobody is using it.

**Comment by osbornk** [comment]
+1

**Pull request merged**